markdown/usage/usage_tutorials.md

```diff
@@ -2,10 +2,19 @@
 title: Usage Tutorials
 subtitle: In-depth step-by-step guides for getting started with nf-core
 ---
+# Table of Content
+  1. [Introduction](#introduction)
+  2. [Adding your Institution to the Community](#AddingyourInstitutiontotheCommunity)
+
 
 # Introduction
 
 This page includes step-by-step guides, walkthroughs, and general long-form introductions to various aspects of nf-core, aimed for brand-new users or developers who are not sure where to start.
 
 - [Walkthrough: A-Z introduction to using and creating nf-core pipelines](tutorials/nf_core_tutorial.md)
   - This walkthrough gives an A-Z overview of how to use and create nf-core pipelines. It is in a workshop format with practical exercises, and gives you all the basic information you need to get yourself up-and-running with getting involved with nf-core.
+
+
+# Adding your Institution to the Community
```

Comment:
This should be a markdown link like above for `Walkthrough: A-Z introduction to using and creating nf-core pipelines`
and the description should be a sub-bullet point